src/main/java/core/basesyntax/UserService.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
+import core.basesyntax.exception.UserNotFoundException;
+
 public class UserService {
 
     public int getUserScore(String[] records, String email) {
-        return 0;
+        for (int i = 0; i < records.length; i++) {
+            if (records[i].substring(0, records[i].indexOf(":")).equals(email)) {
+                email = records[i].substring(records[i].indexOf(":") + 1);
+            }
+        }
+        try {
```

Comment:
if we can't parse int that doesn't mean that user doesn't exist, it means we made some mistake in the logic of the program, let's not use try-catch here, instead, try to split the record with String.split() and save the result to a String[] devidedRecord